app/indexers/hyrax/valkyrie_work_indexer.rb

```diff
@@ -9,6 +9,7 @@ class ValkyrieWorkIndexer < Hyrax::ValkyrieIndexer
     include Hyrax::VisibilityIndexer
     include Hyrax::ThumbnailIndexer
     include Hyrax::Indexer(:core_metadata)
+    include Hyrax::Indexer(:basic_metadata)
```

Comment:
i think this should be avoided, since not all works include basic metadata.

Reply:
@no-reply Without it resource_type doesn't get indexed. Other suggestions on how to do this?

Reply:
my best understanding is:
  - WorkShowPresenter and its callers should tolerate `resource_type` being absent;
  - tests for any (optional) behavior that depends on `resource_type` should use the `:monograph` factory